pom.xml

```diff
@@ -136,9 +136,9 @@
     <shade.prefix>org.apache.pinot.shaded</shade.prefix>
     <shade.phase.prop>none</shade.phase.prop>
 
-    <avro.version>1.11.3</avro.version>
+    <avro.version>1.11.4</avro.version>
```

Comment:
would the `dependabot` bump the version up again?

Reply:
Not sure, but we can tell it to not bump it up when it files the new PR